pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -278,3 +282,53 @@ func createElement(name string, enterpriseID uint32) ipfixentities.InfoElementWi
 	ieWithValue, _ := ipfixentities.DecodeAndCreateInfoElementWithValue(element, nil)
 	return ieWithValue
 }
+
+func TestFlowAggregator_updateFlowAggregator(t *testing.T) {
+
+	dbEP := &clickhouseclient.ClickHouseExportProcess{}
+	opt := &util.Options{}
+	testcases := []struct {
+		FlowCollectorEnable       bool
+		FlowCollectorAddress      string
+		ClickHouseEnable          bool
+		externalFlowCollectorAddr string
+		message                   []updateParam
+		dbExportProcess           *clickhouseclient.ClickHouseExportProcess
+	}{
+		{true, "10.10.10.10:155", false, "", []updateParam{updateExternalFlowCollectorAddr}, nil},
+		{false, "", true, "addr", []updateParam{disableFlowCollector}, nil},
+		{false, "", true, "", []updateParam{enableClickHouse}, nil},
+		{false, "", true, "", []updateParam{updateClickHouseParam}, dbEP},
+		{true, "10.10.10.10:155", false, "", []updateParam{updateExternalFlowCollectorAddr, disableClickHouse}, dbEP},
+	}
+
+	timeout := time.After(1 * time.Minute)
+	done := make(chan bool)
+	go func() {
+		for i, tc := range testcases {
+			t.Run(fmt.Sprintf("subtest: %d", i), func(t *testing.T) {
+				FlowAggregator := &flowAggregator{
+					updateCh: make(chan updateMsg, 2),
```

Comment:
why 2? please add a comment when you hardcode a value like this

Reply:
Removed. The len of channel cannot be check if we don't state the size of channel. I was using that for checking a channel is empty or not.